src/OpenLoco/Localisation/StringManager.cpp

```diff
@@ -517,7 +517,8 @@ namespace OpenLoco::StringManager
                         else
                         {
                             unit = getString(StringIds::unit_kW);
-                            value = std::round(value * 0.746);
+                            constexpr double hpTokW = 764.0 / 1024.0;
+                            value = std::round(value * hpTokW);
```

Comment:
If we are using the integer units there is no need for the round and you can keep it in integer maths

Reply:
I'm no integer-arithmetic wizard but isn't that going to make the calculation something like this?
`value = (value * 191) / 256;`

Reply:
yeah you can divide out 4 if you want.

Reply:
@duncanspumpkin I opted to just multiply by the decimal (do I still need std::round here?), since using integer multiplication here would reduce the valid range of power values by, in this case, 191 times before it overflows the uint32_t.

I don't think this reduction is actually game-limiting, but I also don't see a reason to reduce it unnecessarily.

For fun, I did some calculations:

- The maximum value would be `uint32_t::max() / 191`, or around 22, 486, 739hp.
- The current worlds-most-powerful locomotive is the Shen 24, which has a power of 38,621hp.
- This means the game would be able to string-format 582 of these locos (assuming someone added these in as a custom object) in a single train before overflowing. 
- If people add custom locos with substantially more power, it's conceivable they *may* hit that 22 million hp limit, even though it would require a monstrous train.

Reply:
No the round is not required. Ultimately I only care about keeping everything integer maths in the main game loop. This is ui only so I don't really care.